platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -11,113 +11,162 @@ For details, visit [Azure Free Account][az-create-account].
-13. You will need at least one blob storage container to act as a working directory for Nextflow.
-14. Go to your new storage account and select **+ Container** to create a new Blob storage container. A new container dialogue will open. Enter a suitable name, e.g., _towerrgstorage-container_.
-15. Go to the **Access Keys** section of your new storage account (_towerrgstorage_ in this example).
-16. Store the access keys for your Azure Storage account, to be used when you create a Seqera compute environment.
+1. In **tags**, add any required tags for the storage account.
+1. Select **Review and Create**.
+1. Select **Create** to create the Azure Storage account.
+1. You will need at least one blob storage container to act as a working directory for Nextflow.
+1. Go to your new storage account and select **+ Container** to create a new Blob storage container. A new container dialogue will open. Enter a suitable name, e.g., _towerrgstorage-container_.
+1. Go to the **Access Keys** section of your new storage account (_towerrgstorage_ in this example).
+1. Store the access keys for your Azure Storage account, to be used when you create a Seqera compute environment.
 
 :::caution
 Blob container storage credentials are associated with the Batch pool configuration. Avoid changing these credentials in your Seqera instance after you have created the compute environment.
 :::
 
-## Batch account
-
-After you have created a resource group and storage account, create a [Batch account][az-learn-batch].
+### Batch account
 
-### Create a Batch account
+After you have created a resource group and storage account, create a [Batch account][az-learn-batch]:
 
 1. Log in to your Azure account and select **Create a batch account** on [this page][az-create-batch].
-2. Select the existing resource group or create a new one.
-3. Enter a name for the Batch account, e.g., _towerrgbatch_.
-4. Choose the preferred region (same as the storage account).
-5. Select **Advanced**.
-6. For _Pool allocation mode_, select Batch service.
-7. For _Authentication mode_, ensure _Shared Key_ is selected.
-8. Select **Networking**. Ensure networking access is sufficient for the platform and any additional required resources.
-9. In **tags**, add any required tags for the Batch account.
-10. Select **Review and Create**.
-11. Select **Create**.
-12. Go to your new Batch account, then select **Access Keys**.
-13. Store the access keys for your Azure Batch account, to be used when you create a Seqera compute environment.
-
+1. Select the existing resource group or create a new one.
+1. Enter a name for the Batch account, such as _towerrgbatch_.
+1. Choose the preferred region. This must be the same region as the Storage account.
+1. Select **Advanced**.
+1. For **Pool allocation mode**, select Batch service.
+1. For **Authentication mode**, select _Shared Key_.
+1. Select **Networking**. Ensure networking access is sufficient for Platform and any additional required resources.
```

Comment:
Below you have a guide that has sub points for fields to fill out on a page, would this make sense here also?

For example:

```
In your Platform workspace **Credentials** tab, select the **Add credentials** button and complete the following fields:
```

Reply:
@christopher-hakkaart we could do with some reformatting of most of the instructions lists on this page. In the interests of getting this update out in time for release, I'm going to shelf this suggestion for now, in favor of revisiting when the Compute envs section is tackled as a while during the consistency epic we're tackling soon.